O365WebHook/formatO365Log.js

```diff
@@ -37,10 +37,10 @@ module.exports = function(item) {
         messageType: 'json/azure.o365'
     };
     
-    if (messageTypeId) {
+    if (messageTypeId !== undefined && messageTypeId !== null) {
         formattedMsg.messageTypeId = `${messageTypeId}`;
     }
-    if (creationTime.usec) {
+    if (typeof creationTime.usec === 'number' && !isNaN(creationTime.usec)) {
```

Comment:
I think this is overkill and that is enough to leave `if (creationTime.usec)` check.
Even if `creationTime.usec == 0` we don't need to add the `messageTsUs ` property.